docs/ci-status.rst

```diff
@@ -0,0 +1,79 @@
+CI Status
+=========
+
+This page shows current status of CI jobs that are expected to be stable.
+
+
+Anaconda
+--------
+
+.. |container-autoupdate| image:: https://github.com/rhinstaller/anaconda/actions/workflows/container-autoupdate.yml/badge.svg
+   :alt: Refresh container images
+   :target: https://github.com/rhinstaller/anaconda/actions/workflows/container-autoupdate.yml
+
+.. |container-daily-rhel-copr| image:: https://github.com/rhinstaller/anaconda/actions/workflows/daily-rhel-copr.yml/badge.svg
+   :alt: Build current anaconda rhel-8 branch in RHEL COPR
+   :target: https://github.com/rhinstaller/anaconda/actions/workflows/daily-rhel-copr.yml
+
+.. |tag-release| image:: https://github.com/rhinstaller/anaconda/actions/workflows/tag-release.yml/badge.svg
+   :alt: Release from tags
+   :target: https://github.com/rhinstaller/anaconda/actions/workflows/tag-release.yml
+
+.. _releases: https://github.com/rhinstaller/anaconda/releases
+
+* |container-autoupdate|
```

Comment:
It puts everything on one line. I think it looks better when you remove `* ` to create [definition lists](https://docutils.sourceforge.io/docs/user/rst/quickref.html#definition-lists).

Reply:
Thanks for the idea, changed.